tests/serverless/internal/resources/runtimes/nodejs.go

```diff
@@ -275,8 +275,11 @@ func NodeJSFunctionUsingHanaClient(rtm serverlessv1alpha2.Runtime) serverlessv1a
 				let result = await conn.exec('SELECT 1 AS "One" FROM DUMMY')
 				return result;
 			  } catch(err) {
-				// it is expected to leave here 
-				  return err;
+				// it is expected to leave here. The purpose is to check if hana client returns a known error instead of crashing the whole container with SIGSEGV
+				if(err.sqlState && err.sqlState.startsWith("HY")){
```

Comment:
why the `HY` prefix? I would suggest leaving here comment with details or even linking the hana client documentation

Reply:
done